### PR TITLE
Dw/hero text section

### PR DESCRIPTION
**Pull request by demiwu96**
Adjusted the font size of hero text and footer, clean up some unnecessary code

### CODE REVIEW

File: src/components/AboutUs/TheBottomLineSection.js [hunk 1]
```diff
@@ -104,9 +104,9 @@ const useStyles = makeStyles((theme) => ({
 
 export default function TheBottomLineSection() {
     const classes = useStyles();
-
+ 
     return (
-        <Grid container
+        <Grid container style={{backgroundImage: `url(${People})`}}
```
Comment (rishFilet) [inline src/components/AboutUs/TheBottomLineSection.js]: I don't think we need this change. In my next PR, this is now generic

File: src/components/Footer/Footer.js [hunk 2]
```diff
@@ -81,6 +80,7 @@ const useStyles = makeStyles((theme) => ({
         alignItems: 'center',
         paddingTop: '20px',
         fontSize: '1.5vw',
+        verticalAlign:'middle',
```
Comment (rishFilet) [inline src/components/Footer/Footer.js]: remove spacing
```suggestion

```

File: src/components/Footer/Footer.js [hunk 3]
```diff
@@ -115,20 +116,21 @@ const useStyles = makeStyles((theme) => ({
...
         [theme.breakpoints.down(750)]: {
-            fontSize: '8px',
-            width: '40px',
-            height: '40px'
+            width: '25px',
+            height: '25px',
+            paddingbottom:'5px'
         },
     },
+
```
Comment (rishFilet) [inline src/components/Footer/Footer.js]: remove spacing
```suggestion

```

File: src/components/HeroText/HeroText.module.css [hunk 4]
```diff
@@ -4,18 +4,13 @@
...
-    line-height: 1.8;
-    font-size: calc(0.8em + 0.3vw);
-    font-family: 'Poppins';
-    max-width: 40vw;
+  position: relative;
+  line-height:1.8;
+  font-family: 'Poppins';
 }
 
 .f {
```
Comment (rishFilet) [inline src/components/HeroText/HeroText.module.css]: Change to a more meaningful name

File: src/components/HeroText/HeroText.module.css [hunk 5]
```diff
@@ -31,132 +26,134 @@
...
-    max-width: 100vw;
-    width: 20em;
+    max-width: none;
+    padding-left: 20px;
+    padding-right: 20px;
+    font-size: 14px;
   }
-} */
 
+  .f {
```
Comment (rishFilet) [inline src/components/HeroText/HeroText.module.css]: Change f to a more meaningful name

File: src/components/WhyTechSVG/index.js [hunk 6]
```diff
@@ -1,7 +1,6 @@
 import React from "react";
 import Grid from "@material-ui/core/Grid";
 import Hidden from '@material-ui/core/Hidden';
-import { makeStyles } from "@material-ui/core/styles";
 import ComputerSVG from './ComputerSVG';
```
Comment (rishFilet) [inline src/components/WhyTechSVG/index.js:5]: Add space between package and internal imports

**Review by rishFilet — COMMENTED** [review]
Small spacing changes. Only thing to note is in the bottom line section. In my first of two PRs, i refactored this section to make it more generic. Not sure if we need the background-image here

**Review by hanlinc27 — DISMISSED** [review]
LGTM!

**Comment by gh0stl0nely** [comment]
@demiwu96 Just fix the changes as Rishi suggests + update the branch to meet master' state and we are good to go.

**Comment by rishFilet** [comment]
> Small spacing changes. Only thing to note is in the bottom line section. In my first of two PRs, i refactored this section to make it more generic. Not sure if we need the background-image here


> LGTM!

?

**Comment by gh0stl0nely** [comment]
This is good. Approved.

**Review by gh0stl0nely — DISMISSED** [review]

**Review by hanlinc27 — DISMISSED** [review]

**Comment by gh0stl0nely** [comment]
@demiwu96 Can you update the branch again it seem out of date. And also sorry I missed it but there are some commented out img tags lines in the TimeLineSection.js file, can you also get rid of those while updating the branch? 

image

**Pull request merged**